backends/module.mk

```diff
@@ -166,6 +166,11 @@ MODULE_OBJS += \
 endif
 endif
 
+ifdef USE_STACKTRACES
```

Comment:
Perhaps `USE_STACKTRACE` is a better name?

Reply:
I've removed all the 's' everywhere from stacktraces to stacktrace